src/convertToObject.js

```diff
@@ -1,18 +1,49 @@
 'use strict';
 
+// const stylesString = require('./stylesString');
+
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splited = sourceString.split(';');
```

Comment:
- your solution look like a bit complicated, let's try to improve it )
 - you can overwrite this code with only one cycle for
- please check this solution

<img width="630" alt="Screenshot 2023-09-27 at 13 10 12" src="https://github.com/mate-academy/js_style-to-object/assets/64746943/0b53f858-b1f5-4bda-8a0d-fd89341b1083">